build.sbt

```diff
@@ -1,10 +1,23 @@
+val circeVersion = "0.9.3"
+
 lazy val root = project
   .in(file("."))
   .settings(
     name := "rock-paper-scissor",
     scalaVersion := "2.12.3",
     resolvers += "buildo at bintray" at "https://dl.bintray.com/buildo/maven",
+    resolvers += Resolver.bintrayRepo("hseeberger", "maven"),
```

Comment:
was this needed? Strange.

Reply:
I didn't copy the dependencies from the solution because I wanted to get a better understanding of how it worked, so I just followed the instructions here https://github.com/hseeberger/akka-http-json without questioning them 🙄

Reply:
Right, as he says in the comment final versions are also published on Maven, so it should be available without a custom resolver.